galasa-inttests-parent/dev.galasa.inttests/src/main/java/dev/galasa/inttests/compilation/simbank/AbstractCompilationLocalSimBank.java

```diff
@@ -199,5 +203,13 @@ private void changePrefix(Path file) throws IOException {
         Files.write(file, fileData.getBytes());
         logger.trace("Changing prefix (" + incumbent + ") to \"" + prefix + "\" in file: " + file.toString());
     }
+
+    private void makeGradleProperties() throws IOException{
+        String gradleProperties = "gradle.properties";
+        String obr = "sourceMaven=https://development.galasa.dev/main/maven-repo/obr/";
+        Path gradlePropertiesFile = testRunDirectory.resolve("/.gradle/" + gradleProperties);
```

Comment:
Move this same logic into a different method called createParentGradleProperties as mentioned in my last comment.